test/expected.txt

```diff
@@ -547,15 +547,15 @@ FAIL	/scroll-animations/css/view-timeline-shorthand.html	Shorthand contraction o
 FAIL	/scroll-animations/css/view-timeline-with-delay-and-range.tentative.html	ViewTimeline with animation delays and range
 FAIL	/scroll-animations/css/view-timeline-with-transform-on-subject.html	ViewTimeline use untransformed box for range calculations
 PASS	/scroll-animations/scroll-timelines/cancel-animation.html	Canceling an animation should cause its start time and hold time to be unresolved
-PASS	/scroll-animations/scroll-timelines/cancel-animation.html	A play-pending ready promise should be rejected when the animation is canceled
+FAIL	/scroll-animations/scroll-timelines/cancel-animation.html	A play-pending ready promise should be rejected when the animation is canceled
```

Comment:
Adding the current results here for comparison. Will look into the failing tests.